widget/importance.go

```diff
@@ -0,0 +1,24 @@
+package widget
+
+type Importance int
+
+const (
+	// MediumImportance applies a standard appearance.
+	MediumImportance Importance = iota
+	// HighImportance applies a prominent appearance.
+	HighImportance
+	// LowImportance applies a subtle appearance.
+	LowImportance
+
+	// DangerImportance applies an error theme to the widget.
+	//
+	// Since 2.3
+	DangerImportance
+	// WarningImportance applies a warning theme to the widget.
+	//
+	// Since 2.3
+	WarningImportance
+
+	// SuccessImportance applies a success theme to the widget.
+	SuccessImportance
```

Comment:
Hmm. I'm not sure about this as an importance. Success does not seem like an importance level to me. Can we leave that out for now and focus on adding importance support to label?

Reply:
I will need the possibility to print green (success theme) message. Don't know how it can be called, but it's usefull to print text like "successfully done something" or "up-to-date something". In my case, it will be to print positive value like +200. Negative are printed in red (DangerImportance)

Reply:
Maybe HighImportance on label print in Success color instead of Primary color ?

Reply:
We should not be inconsistent - high importance does not mean success.

I think that adding success here makes sense - the naming may not be optimal, but we have been using it on button since last release so why not go with it?

Reply:
Have we been using it on Button since last release? We didn't have it as an importance before as far as I can see

Reply:
I agree that success in itself is good to add but yeah, not sure about naming there. I can let it pass if you're happy with it

Reply:
Success probably is the better one in that case. It is what we call the colour

Reply:
> Have we been using it on Button since last release? We didn't have it as an importance before as far as I can see

My apologies there was not a success button before. I kindof assumed there was!
I'm not too bothered one way or another but if the purpose is to expose the meaning here in general I don't think there's a reason to leave off success.

Reply:
Agreed. Let's go for that then. Sorry for the back and forth @blaize

Reply:
Ok no problem. So I will add SuccessImportance. Should I have to also patch the Button widget to have it react to SuccessImportance ?

Reply:
Sorry for the late reply. I see you added it. Thanks :)